pkg/detectors/gitlabv2/gitlab.go

```diff
@@ -42,7 +46,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 			continue
 		}
 
-		secret := detectors.Result{
+		s1 := detectors.Result{
```

Comment:
same question as v1 rename

Reply:
yes, i updated this for any detectors i touched

Reply:
there's a few other detectors that still use `secret`. i was planning on updating them after all the tristate detectors have been updated (if they don't get updated during the process).